### ISSUE TITLE
MaterialComboBox cannot refresh Datasource programmatically

### ISSUE THREAD

**Issue by Pasha-gharibi** [GwtMaterialDesign/gwt-material-addins#199]
I am using a MaterialCombobox which I want to refresh its data programmatically, but previous data remains in it and new data appends under the previous data, how can I resolve it?

```
	@UiField
	MaterialComboBox<FilterItem> topics;
```

I have checked these, but not work.

```
		this.topics.getValues().clear();
		this.topics.setValues(null);
		this.topics.setValues(new ArrayList<FilterItem>());
```

Thanks,

**Comment by Pasha-gharibi** [comment]
I resolved this by this line of code:

```
		this.getValues().clear();
		this.getListbox().clear();
```

**Comment by kevzlou7979** [comment]
This has been solved in 2.0-SNAPSHOT, Try to call combobox.clear() as it clears also the listbox and it's values. See the code implementation here https://github.com/GwtMaterialDesign/gwt-material-addins/blob/release_2.0/src/main/java/gwt/material/design/addins/client/combobox/MaterialComboBox.java#L506

**Comment by Pasha-gharibi** [comment]
Thanks for your reply
I am already using gwt-material-2.0-rc3.jar
Let me check it
